src/error.rs

```diff
@@ -155,17 +162,17 @@ impl From<LalrpopParseError<usize, Token<'_>, &str>> for ParseErrorMsg {
                 help: String::new(),
             },
             LalrpopParseError::User { error } => ParseErrorMsg {
```

Comment:
Can't you just return `error` directly rather than creating a new ParseErrorMsg that is an exact copy?